package/src/countmon.cpp

```diff
@@ -10,6 +10,7 @@
 #include <iostream>
 #include <sstream>
 
+#include "MessageBase.h"
```

Comment:
We don't really need to include this header in the source files since it's already in the header, no?